externalTriBITSPamgem.patch

```diff
@@ -0,0 +1,78 @@
+diff --git a/TPLsList.cmake b/TPLsList.cmake
+index eab14e1676..8b38e12a75 100644
+--- a/TPLsList.cmake
++++ b/TPLsList.cmake
+@@ -68,7 +68,7 @@ SET( Seacas_TPLS_FINDMODS_CLASSIFICATIONS
+   DataWarp        "cmake/TPLs/"    SS
+   METIS           "cmake/TPLs/"    SS
+   ParMETIS        "cmake/TPLs/"    SS
+-  Pamgen          "cmake/TPLs/"    SS
++  Pamgen          "TRIBITS_PKG"    SS
```

Comment:
@bartlettroscoe Does this change allow a non-Trilinos build of SEACAS to find an externally installed (perhaps via Trilinos) version of PamGen?

Reply:
Hi Greg, I don't know if it could support that in all cases since it relies on a TriBITS externally built Pamgen which is enabled by passing `Trilinos_ENABLE_Pamgen=ON`.

Reply:
Most likely the Pamgen would be TriBITs installed, so that should work...  Even if installed with spack, it is supplied by Trilinos.

Should be OK then.